docs/arm64e-Deployment.md

```diff
@@ -12,9 +12,9 @@ For more information on pointer authentication, refer to the [official documenta
 ## Building for iOS 12.0–13.7
 You may have been directed here by the following warning output by Theos:
 
-> **Warning:** Building for iOS 7.0, but the current toolchain can’t produce arm64e binaries for iOS earlier than 14.0. More information: https://github.com/theos/theos/wiki/arm64e-Deployment
+> **Warning:** Building for iOS 7.0, but the current toolchain can’t produce arm64e binaries for iOS earlier than 14.0. More information: https://theos.dev/docs/arm64e-deployment
```

Comment:
It can be, but I just copied the error message 1:1 from [Theos](https://github.com/theos/theos/blob/a12c440f566afad7e0f1578b2414fd5c8ab899e5/makefiles/targets/_common/iphone.mk#L95).

Reply:
ah, I wasn't paying attention to the context. Makes sense, sounds good.